hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/customrenderers/client/renderers/LinkRenderer.java

```diff
@@ -21,11 +24,21 @@
 public class LinkRenderer extends ButtonRenderer {
     @Override
     public void render(RendererCellReference cell, String text, Button button) {
-        button.setText(text);
+        Map<String, String> nameStatusMap = formatData(text);
```

Comment:
I'm not sure what is in the `text` value to split it up into a map to then retrieve the `name` and the `status` is this Vaadin related? Can this maybe documented or is this documented?